extensions/notebook/src/dialog/managePackages/addNewPackageTab.ts

```diff
@@ -48,6 +49,14 @@ export class AddNewPackageTab {
 				await this.loadNewPackageInfo();
 			});
 
+			this.installProgressSpinner = view.modelBuilder.loadingComponent()
+				.withProps({
+					loadingText: localize('managePackages.installProgressText', "Package installation in progress"),
```

Comment:
Personal preference, but I like `Installing package`/`Package installed` better - shorter is generally preferable